docs/claude/architecture.md

```
@@ -0,0 +1,58 @@
+
+| Collection | Source | Loader |
+|------------|--------|--------|
+| **post** | `src/data/post/` | Local MDX via glob loader |
+| **til** | `src/content/til/` | "Today I Learned" markdown |
+| **resources** | Notion database | Vendored `notion-astro-loader` (filters Status = "Up-to-Date") |
+
+The vendored loader in `vendor/notion-astro-loader/` is a local fork. Modify there for Notion integration changes.
+
+## Component Organization
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)